### PR TITLE
Feature/sig 3 register user resticctions

### PR DESCRIPTION
**Pull request by MatiiBr**

### CODE REVIEW

File: src/app/views/login/login.page.ts [hunk 1]
```diff
@@ -45,12 +52,7 @@ import { AuthenticationService } from 'src/app/services/authentication/authentic
               formControlName="email"
               placeholder="Email"
             ></ion-input>
-            <ion-input
-              class="ui-form-input"
-              formControlName="password"
-              placeholder="Contraseña"
-              type="password"
-            ></ion-input>
+            <app-password-input controlName="password"> </app-password-input>
```
Comment (pedro-mar99) [inline src/app/views/login/login.page.ts]: Falta el placeholder :)

File: src/app/views/login/login.page.ts [hunk 2]
```diff
@@ -1,6 +1,13 @@
-import { Component, OnInit } from '@angular/core';
+import {
```
Comment (pedro-mar99) [inline src/app/views/login/login.page.ts]: Revisar import, no se están usando

**Review by pedro-mar99 — CHANGES_REQUESTED** [review]

File: src/app/views/login/login.page.ts (hunk 1 above)
Comment (MatiiBr) [reply src/app/views/login/login.page.ts]: Ready

File: src/app/views/login/login.page.ts (hunk 2 above)
Comment (MatiiBr) [reply src/app/views/login/login.page.ts]: Ready

**Review by pedro-mar99 — APPROVED** [review]

**Pull request merged**